guides/source/active_record_basics.md

```diff
@@ -71,88 +67,105 @@ Convention over Configuration in Active Record
+some naming and schema conventions to follow. And in case you need to, it is possible to [override naming conventions](#overriding-the-naming-conventions).
 
 ### Naming Conventions
 
-By default, Active Record uses some naming conventions to find out how the
-mapping between models and database tables should be created. Rails will
-pluralize your class names to find the respective database table. So, for
-a class `Book`, you should have a database table called **books**. The Rails
-pluralization mechanisms are very powerful, being capable of pluralizing (and
-singularizing) both regular and irregular words. When using class names composed
-of two or more words, the model class name should follow the Ruby conventions,
-using the CamelCase form, while the table name must use the snake_case form. Examples:
+Active Record uses this naming convention to map between models (represented by
+Ruby objects) and database tables:
+
+Rails will pluralize your model's class names to find the respective database
+table. For example, a class named `Book` maps to a database table named
+**books**. The Rails pluralization mechanisms are very powerful and capable of
+pluralizing (and singularizing) both regular and irregular words. This uses the
+[Active Support](active_support_core_extensions.html#pluralize) [pluralize
+method](https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-pluralize),
+if you're curious.
+
+For class names composed of two or more words, the model class name will follow
+the Ruby conventions of using a CamelCase name. The database table name, in that
+case, will be a snake_case name. For example:
 
-* Model Class - Singular with the first letter of each word capitalized (e.g., `BookClub`).
-* Database Table - Plural with underscores separating words (e.g., `book_clubs`).
+* `BookClub` - is the model class, singular with the first letter of each word capitalized
+* `book_clubs` - is the matching database table, plural with underscores separating words
+
+Here are some more examples of model class names and corrosponding table names:
 
 | Model / Class    | Table / Schema |
 | ---------------- | -------------- |
 | `Article`        | `articles`     |
 | `LineItem`       | `line_items`   |
-| `Deer`           | `deers`        |
+| `Product`        | `products`     |
 | `Mouse`          | `mice`         |
 | `Person`         | `people`       |
 
 ### Schema Conventions
 
-Active Record uses naming conventions for the columns in database tables,
+Active Record uses conventions for column names in the database tables as well,
 depending on the purpose of these columns.
 
+* **Primary keys** - By default, Active Record will use an integer column named
+  `id` as the table's primary key (`bigint` for PostgreSQL and MySQL, `integer`
+  for SQLite). When using [Active Record
+  Migrations](active_record_migrations.html) to create your tables, this column
```

Comment:
Let's maybe link to the migration section within the guide, and let that section point to the separate migrations guide.